app/posts/personalised-prevention-platform/2025/03/2025-03-31-technical-research.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Technology research summary"
+description: A summary of technology within NHSE that could support personalised prevention work
+date: 2025-03-31
+tags:
+- discovery
+---
+
+As part of the Personalised Prevention Platform discovery, we examined the existing and planned NHS England technology platforms and products so we could understand what could be reused or extended, and what we might need to build from scratch.
+
+## Problem statement
+
+We need to understand the availability and readiness of the NHS platforms and data we could use to identify and select people with certain characteristics and risk factors.
+
+Our current problem statement will continue to evolve over the course of discovery and alpha as we learn more about the problem we are trying to understand, and the hypotheses we are aiming to prove or disprove.
```

Comment:
Could / should we match the other post?

> Our current problem statement has evolved over the course of discovery as we learned more about the problem we were trying to understand. It may change as we learn more in the alpha.